runtime/pendulum/src/xcm_config.rs

```diff
@@ -95,6 +99,9 @@ impl Convert<MultiLocation, Option<CurrencyId>> for CurrencyIdConvert {
 						GeneralIndex(asset_hub::USDT_ASSET_ID),
 					),
 			} => Some(CurrencyId::XCM(XCM_ASSET_ASSETHUB_USDT)),
+			MultiLocation { parents: 1, interior: X2(Parachain(id), PalletInstance(10)) }
```

Comment:
Are we confident that this is correct (is it defined this way on other chains or do the XCM docs specify that we need to do it this way?).

MultiLocations are always relative and I know that they are usually transformed when they occur in an XCM messages that is sent to another chain so that the MultiLocation that made sense in the context of the sender of the parachain will then look different and will be adapted to the context of the target parachain where it is executed.

In this case I think that an incoming XCM messages that is sent to us will actually have a multilocation that will be transformed to look like `MultiLocation { parents: 0, interior: X1(PalletInstance(10)) }`.

I could be totally wrong, though.

Reply:
You're right, I also added `MultiLocation { parents: 0, interior: X1(PalletInstance(10)) }` because it is needed when receiving the native token via XCM. Sender uses `MultiLocation { parents: 1, interior: X2(Parachain(id), PalletInstance(10)) }` but this gets transformed on the receiving side before being converted to CurrencyID.